kikimay/store-manager-Api
# # 161203566 write failing tests for the endpoints

**What does this PR do?**
It creates failing tests for testing the API endpoints.
 **Description of Task to be completed?**
write tests for each endpoint and run them.
**How should this be manually tested?**
clone the repo into a desired folder.cd into it and open the folder with  your favorite text editor run the tests in your terminal to see they fail because there no code for endpoints yet
 **Any background context you want to provide?**
none
 **What are the relevant pivotal tracker stories?**
161203566 
**Screenshots (if appropriate)**
![screenshot from 2018-10-17 09-32-41](https://user-images.githubusercontent.com/39055415/47066717-a3c84480-d1ef-11e8-9ed2-58bd999c5e3f.png)
